src/pages/content/grammar.mdx

```diff
@@ -1,12 +1,145 @@
 ---
 title: Grammar
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Acronyms and abbreviations
+
+Acronyms are like inside jokes. If you aren’t in the know they don’t make sense. Use acronyms to
+save space, but not at the risk of confusing people. Spell out an acronym on first use and include
+it in parentheses. ​
+
+_A service level agreement (SLA) is a contract between you and your customers._
+
+### Bold and italics
+
+Use bold to refer to UI elements. Avoid italics or quotation marks. Using bold is a bold statement.
+A little bit goes a long way, so use bold sparingly. ​ Don't get too bold in notifications. Only use
+bold for headings, and not in the description.
```

Comment:
Is it worth incorporating the technicality that Garden's bold weight is semibold (600)? Might be the wrong place to call this out.

Reply:
I don't know if this is the right spot for it.